README.md

```diff
@@ -109,11 +109,13 @@ with:
   userFraction: 0.33
   whatsNewDirectory: distribution/whatsnew
   mappingFile: app/build/outputs/mapping/release/mapping.txt
+  status: draft
+  changesNotSentForReview: true
```

Comment:
This doesn't make sense as an example usage. Draft shouldn't include userFraction as well. The draft should be a separate example.